src/StreamRequest.jl

```diff
@@ -151,4 +152,13 @@ function readbody(http::Stream, res::Response, response_stream, reached_redirect
     end
 end
 
+function save_response_stream!(response::Response, response_stream)
+    resp = UInt8[]
+    response_stream_copy = deepcopy(response_stream)
```

Comment:
why the `deepcopy` here?

Reply:
I didn't want to alter `response_stream` in case it's used elsewhere. But I don't think it is, so could be safe to remove